src/olympia/shelves/models.py

```diff
@@ -29,9 +33,12 @@ class Shelf(ModelBase):
     )
     addon_count = models.PositiveSmallIntegerField(
         default=0,
-        help_text='0 means the default number (4, or 3 for search-themes) of add-ons '
+        help_text='0 means the default number (4, or 3 for themes) of add-ons '
         'will be included in shelf responses. Set to override.',
     )
+    addon_type = models.CharField(
+        max_length=200, choices=ADDON_TYPE_CHOICES, default='extensions'
```

Comment:
To be consistent, this should be the same addon type we use for `Addon` (it's an int).